packages/block-library/src/latest-posts/edit.js

```diff
@@ -104,8 +118,9 @@ class LatestPostsEdit extends Component {
 			featuredImageSizeSlug,
 			featuredImageSizeWidth,
 			featuredImageSizeHeight,
+			postType,
 		} = attributes;
-		const suggestions = categoriesList.reduce(
+		const categorySuggestions = categoriesList.reduce(
```

Comment:
you changed this here from [my comment](https://github.com/WordPress/gutenberg/pull/20832#discussion_r391647710) in the other PR right?